chia/server/chia_policy.py

```diff
@@ -275,7 +275,7 @@ async def _chia_accept_loop(self, listener: socket.socket) -> Tuple[socket.socke
 
         def _chia_accept(self, listener: socket.socket) -> asyncio.Future[Tuple[socket.socket, Tuple[object, ...]]]:
             self._register_with_iocp(listener)
-            conn = self._get_accept_socket(listener.family)
+            conn = self._get_accept_socket(listener.family)  # pylint: disable=assignment-from-no-return
```

Comment:
is mypy mistaken in thinking that `_get_accept_socket()` doesn't return anything?

Reply:
pylint is the confused one.  It doesn't special case the `typing.TYPE_CHECKING` afaik so would treat it as the `False` it is coded to for runtime.  This means it misses all of our hinting shims above.  There may be more to look at here.